tracie/data_public/README.md

```diff
@@ -0,0 +1,12 @@
+This directory contains the files `train.jsonl`, `dev.jsonl` and `test.jsonl`. Please note that `train.jsonl` and `dev.jsonl` are the same file, since we do not have an explicit `dev` split (for our experiments, we use a portion of the training set to tune models on `train.jsonl`).
```

Comment:
If train.jsonl and dev.jsonl are the same, and dev.jsonl isn't used anywhere in this code, can we just keep train.jsonl (which is used)?

Reply:
yeh, that is fine.

Reply:
can you go ahead and remove the dev file then?